### PR TITLE
Added downloadsToolbars labels to rendererIdentifiers array

### PR DESCRIPTION
**Pull request by luixxiul**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

Addresses #3166

### CODE REVIEW

**Comment by bbondy** [comment]
well is it needed or not? It was not clear to me. If it's not needed we shouldn't pull this in.

**Comment by luixxiul** [comment]
In the other PRs I have added ones, so I think we need these in this case too. If it works without them, I will close this.

**Comment by bbondy** [comment]
screenshot 2016-08-14 10 33 40

**Comment by bbondy** [comment]
...so merging

File: app/locale.js [hunk 1]
```diff
@@ -179,7 +179,17 @@ var rendererIdentifiers = function () {
     'yes',
     'no',
     'neverForThisSite',
-    'passwordsManager'
+    'passwordsManager',
+    'downloadItemPause',
+    'downloadItemResume',
+    'downloadItemCance',
```
Comment (bbondy) [inline app/locale.js]: this should be `downloadItemCancel`

**Comment by bbondy** [comment]
++ thanks!

**Pull request merged**